contrib/intel/jenkins/build.py

```diff
@@ -186,13 +188,15 @@ def log_dir(install_path, release=False):
     libfab_install_path = f'{cloudbees_config.install_dir}/{jobname}/{buildno}/{ofi_build_mode}'
 
     if (ucx):
-        libfab_install_path += "/ucx"
+        libfab_install_path += '/ucx'
+        workspace += '/ucx'
 
     p = re.compile('mpi*')
 
     if (build_item == 'libfabric'):
             build_libfabric(libfab_install_path, ofi_build_mode, cluster, ucx)
     elif (build_item == 'libfabric_mpich'):
+            workspace += '/mpich'
```

Comment:
don't we need to change the dir to workspace while building libfabric?

Reply:
this was here because ucx needed it for fabtests. Since this build doesn't make fabtests i'll remove it